tests/kafkatest/services/kafka/kafka.py

```diff
@@ -229,7 +229,8 @@ def start_node(self, node):
         self.logger.debug("Attempting to start KafkaService on %s with command: %s" % (str(node.account), cmd))
         with node.account.monitor_log(KafkaService.STDOUT_STDERR_CAPTURE) as monitor:
             node.account.ssh(cmd)
-            monitor.wait_until("Kafka Server.*started", timeout_sec=30, backoff_sec=.25, err_msg="Kafka server didn't finish startup")
+            # Kafka 1.0.0 and higher don't have a space between "Kafka" and "Server"
```

Comment:
Would this be better as an `nc -z` test? Grepping logs was always kind of a half-assed solution, and the real test we care about seems to be whether anything is actually listening on the port.

We have this now for at least jmx (thanks to yours truly) and ZK (thanks to @kkonstantine) and I'd like to continue the trend elsewhere as it is far less brittle than grepping logs.

Reply:
Would it be OK if I submit a follow-up with the more robust fix? Without this change, every system test involving Kafka is broken.